x-pack/plugins/ml/server/models/data_recognizer/modules/security_windows/ml/v3_windows_anomalous_script.json

```diff
@@ -0,0 +1,53 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "Looks for unusual powershell scripts that may indicate execution of malware, or persistence mechanisms.",
```

Comment:
To be consistent with all the other descriptions in this module, should add `Security: Windows - ` onto the front of the text here.

Reply:
done